### ISSUE TITLE
Unable to enable the S.M.A.R.T in openmediavault

### ISSUE THREAD

**Issue by huangwb8** [openmediavault/openmediavault#627]
### Description of issue/question
When I want to enable SMART in openmediavault, I got an error information like that:
```
Error #0:
OMV\ExecException: Failed to execute command 'export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin; export LANG=C.UTF-8; omv-salt deploy run smartmontools 2>&1' with exit code '1': debian:
----------
          ID: configure_default_smartmontools
    Function: file.managed
        Name: /etc/default/smartmontools
      Result: True
     Comment: File /etc/default/smartmontools updated
     Started: 19:56:52.292191
    Duration: 194.067 ms
     Changes:
              ----------
              diff:
                  ---
                  +++
                  @@ -2,5 +2,5 @@
                   # WARNING: Do not edit this file, your changes will get lost.

                   # Defaults for smartmontools initscript (/etc/init.d/smartmontools)
                  -# This is a POSIX shell fragment#start_smartd=yes
                  +# This is a POSIX shell fragmentstart_smartd=yes
                   smartd_opts="--quit=never --interval=1800"
----------
          ID: configure_smartd_conf
    Function: file.managed
        Name: /etc/smartd.conf
      Result: True
     Comment: File /etc/smartd.conf is in the correct state
     Started: 19:56:52.486641
    Duration: 413.154 ms
     Changes:
----------
          ID: enable_smart_on_/dev/disk/by-id/ata-WDC_WD40EFRX-68WT0N0_WD-WCC4E0XASCEL
    Function: cmd.run
        Name: export LANG=C; smartctl -s on /dev/disk/by-id/ata-WDC_WD40EFRX-68WT0N0_WD-WCC4E0XASCEL
      Result: True
     Comment: onlyif condition is false
     Started: 19:56:52.911141
    Duration: 175.534 ms
     Changes:
----------
          ID: enable_smart_on_/dev/disk/by-id/ata-WDC_WD40EFRX-68EUZN0_WD-WCC7K2PC3354
    Function: cmd.run
        Name: export LANG=C; smartctl -s on /dev/disk/by-id/ata-WDC_WD40EFRX-68EUZN0_WD-WCC7K2PC3354
      Result: True
     Comment: onlyif condition is false
     Started: 19:56:53.087577
    Duration: 124.944 ms
     Changes:
----------
          ID: enable_smart_on_/dev/disk/by-id/ata-WDC_WD40EFRX-68EUZN0_WD-WCC7K2PC3357
    Function: cmd.run
        Name: export LANG=C; smartctl -s on /dev/disk/by-id/ata-WDC_WD40EFRX-68EUZN0_WD-WCC7K2PC3357
      Result: True
     Comment: onlyif condition is false
     Started: 19:56:53.214119
    Duration: 515.971 ms
     Changes:
----------
          ID: enable_smart_on_/dev/disk/by-id/ata-WDC_WD40EFRX-68N32N0_WD-WCC7K0ENNP9Y
    Function: cmd.run
        Name: export LANG=C; smartctl -s on /dev/disk/by-id/ata-WDC_WD40EFRX-68N32N0_WD-WCC7K0ENNP9Y
      Result: True
     Comment: onlyif condition is false
     Started: 19:56:53.731506
    Duration: 225.551 ms
     Changes:
----------
          ID: start_smartd_service
    Function: service.running
        Name: smartd
      Result: False
     Comment: The named service smartd is not available
     Started: 19:56:54.039978
    Duration: 77.457 ms
     Changes:

Summary for debian
------------
Succeeded: 6 (changed=1)
Failed:    1
------------
Total states run:     7
Total run time:   1.727 s in /usr/share/php/openmediavault/system/process.inc:182
Stack trace:
#0 /usr/share/php/openmediavault/engine/module/serviceabstract.inc(60): OMV\System\Process->execute()
#1 /usr/share/openmediavault/engined/rpc/config.inc(167): OMV\Engine\Module\ServiceAbstract->deploy()
#2 [internal function]: Engined\Rpc\Config->applyChanges(Array, Array)
#3 /usr/share/php/openmediavault/rpc/serviceabstract.inc(123): call_user_func_array(Array, Array)
#4 /usr/share/php/openmediavault/rpc/serviceabstract.inc(149): OMV\Rpc\ServiceAbstract->callMethod('applyChanges', Array, Array)
#5 /usr/share/php/openmediavault/rpc/serviceabstract.inc(588): OMV\Rpc\ServiceAbstract->OMV\Rpc\{closure}('/tmp/bgstatusmV...', '/tmp/bgoutputYA...')
#6 /usr/share/php/openmediavault/rpc/serviceabstract.inc(159): OMV\Rpc\ServiceAbstract->execBgProc(Object(Closure))
#7 /usr/share/openmediavault/engined/rpc/config.inc(189): OMV\Rpc\ServiceAbstract->callMethodBg('applyChanges', Array, Array)
#8 [internal function]: Engined\Rpc\Config->applyChangesBg(Array, Array)
#9 /usr/share/php/openmediavault/rpc/serviceabstract.inc(123): call_user_func_array(Array, Array)
#10 /usr/share/php/openmediavault/rpc/rpc.inc(86): OMV\Rpc\ServiceAbstract->callMethod('applyChangesBg', Array, Array)
#11 /usr/sbin/omv-engined(537): OMV\Rpc\Rpc::call('Config', 'applyChangesBg', Array, Array, 1)
#12 {main}
```
It seems like the problem is here
```
 ID: start_smartd_service
    Function: service.running
        Name: smartd
      Result: False
     Comment: The named service smartd is not available
     Started: 19:56:54.039978
    Duration: 77.457 ms
     Changes:
```
but how to fixed it? Thanks!
### Steps to reproduce issue
(Include debug logs if possible and relevant.)
+ open the web interface of openmediavault
+ click Storage--Settings--Enable--apply
+ error appears and the change not saved

### Versions report
+ Linux version 5.3.18-2-pve (build@pve) (gcc version 8.3.0 (Debian 8.3.0-6)) #1 SMP PVE 5.3.18-2 (Sat, 15 Feb 2020 15:11:52 +0100)
+ omv 5.3.2-1
+ Use ZFS to manage device

**Comment by votdev** [comment]
I'm not sure if this setting is still in use because Debian 10 uses systemd and the variable `start_smartd` is only used by the SysV init script.

**Comment by huangwb8** [comment]
@votdev All right, hope more proper method to solve the problem! So, it is temtatively no-solution, isn't it?

**Comment by votdev** [comment]
Ah i see the problem, you are using a backport repository which installs a smartmontools package where the unit file has been renamed. This is not supported in OMV in general because backports introduce new versions which may differ from the stable version. I would close the issue because of that reason, but i see that the smartmontools in Buster already contains the `/lib/systemd/system/smartmontools.service` unit file.

**Comment by votdev** [comment]
Fixed in openmediavault 5.3.5.
